components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/common/OneTimeTriggerInboundRunner.java

```diff
@@ -50,16 +49,6 @@ public void run() {
             if (log.isDebugEnabled()) {
                 log.debug("Waiting for the configuration context to be loaded to run Inbound Endpoint.");
             }
-            Boolean isSinglNode = ClusteringAgentUtil.isSingleNode();
-            if (isSinglNode != null) {
-                if (!isSinglNode && !CarbonUtils.isWorkerNode()) {
-                    // Given node is the manager in the cluster, and not
-                    // required to run the service
-                    execute = false;
-                    log.info("Inbound Endpoint will not run in manager node. Same will run on worker(s).");
-                }
-                init = true;
```

Comment:
so we never exit from the loop?

Reply:
mistake on my part. Will remove the loop since we are removing carbon.mediation.clustering from mediation